README.md

```diff
@@ -5,44 +5,54 @@ Pontoon is a translation management system used and developed by the
 specializes in open source localization that is driven by the community and
 uses version-control systems for storing translations.
 
-Check out our documentation to learn how to:
+[📚 **Documentation**](https://mozilla-pontoon.readthedocs.io/)
 
-* [Localize with Pontoon](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/)
-* [Make your projects localizable with Pontoon](https://mozilla-pontoon.readthedocs.io/en/latest/user/localizing-your-projects.html)
 
-## Installation
+## Installing Pontoon
 
-Our [installation documentation](https://mozilla-pontoon.readthedocs.io/) is available on `Read the Docs`:
-
-* For local development, see [developer setup](http://mozilla-pontoon.readthedocs.io/en/latest/dev/setup.html) using `Docker`
-* For production installation, see [deployment documentation](http://mozilla-pontoon.readthedocs.io/en/latest/admin/deployment.html)
-* For quick and easy deployment on the `Heroku` platform, click this button:
+If you are looking to host your own instance of Pontoon, there are several ways to do so.
+The simplest one is to use the **Heroku platform**, and simply click this button:
```

Comment:
I had to re-read this sentence twice to understand it (because of the grammar in the part following comma - even though it seems correct). Maybe replace it with something like this?
```
To deploy Pontoon to Heroku without leaving the web browser, click the **Deploy to Heroku** button below.
```